pkg/agent/ipassigner/ip_assigner_linux.go

```diff
@@ -495,6 +495,7 @@ func (a *ipAssigner) getAssignee(subnetInfo *crdv1b1.SubnetInfo, createIfNotExis
 		LinkAttrs: netlink.LinkAttrs{
 			Name:        name,
 			ParentIndex: a.externalInterface.Index,
+			PermHWAddr:  util.GenerateRandomMAC(),
```

Comment:
@xliuxu I noticed `ServiceExternalIP` feature also use AssignIP, please take a look if there is any impact to `ServiceExternalIP`  if we have a random mac for the vlan sub-interfaces.